tests/lib/nested.sh

```diff
@@ -1270,10 +1291,10 @@ nested_start_classic_vm() {
     local PARAM_DISPLAY PARAM_NETWORK PARAM_MONITOR PARAM_USB PARAM_CPU PARAM_CD PARAM_RANDOM PARAM_SNAPSHOT
     PARAM_DISPLAY="-nographic"
     PARAM_NETWORK="-net nic,model=virtio -net user,hostfwd=tcp::$NESTED_SSH_PORT-:22"
-    PARAM_MONITOR="-monitor tcp:127.0.0.1:$NESTED_MON_PORT,server,nowait"
+    PARAM_MONITOR="-monitor tcp:127.0.0.1:$NESTED_MON_PORT,server=on,wait=off"
     PARAM_USB="-usb"
     PARAM_CPU=""
-    PARAM_CD="${NESTED_PARAM_CD:-}"
+    PARAM_CD="${NESTED_PARAM_CD:-}"on
```

Comment:
this seems wrong, it could be empty in which case this is just "on" ?

Reply:
yes, thanks